configs/audio_recognition/README.md

```diff
@@ -0,0 +1,55 @@
+# Audio Recognition
```

Comment:
the folder structure does not align with other tasks. how about using this `configs/audio/recognition`?
do you have better proposals for the name `recognition`?

Reply:
or `configs/modality/audio`?

Reply:
I think `configs/recognition_audio/resnet/xxx.py` is better. It is consist with `task/model/xx.py` as rgb recognition. `configs/modality/audio` is confusing, as we used to call 'flow' as a modality, but they are mixed up in rgb recognition configs.

Reply:
how about `configs/audio/resnet/xxx.py`?

Reply:
fine let's keep `recognition_audio`